modules/fluid_properties/include/userobjects/NaKFluidProperties.h

```diff
@@ -0,0 +1,86 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "SinglePhaseFluidProperties.h"
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Woverloaded-virtual"
+
+/**
+ * NaK fluid properties as a function of pressure (Pa) and temperature (K).
+ *
+ * Origin of fluid properties:
+ * 1) density; thermal diffusivity; dynamic viscosity; specific heat capacity
+ *
+ * SODIUM-NaK ENGINEERING HANDBOOK Volume I
+ * Sodium Chemistry and Physical Properties
+ * O. J. FOUST, Editor
+ * ISBN 0 677 03020 4
+ */
+class NaKFluidProperties : public SinglePhaseFluidProperties
+{
+public:
+  static InputParameters validParams();
+
+  NaKFluidProperties(const InputParameters & parameters);
+  virtual ~NaKFluidProperties();
+
+  virtual std::string fluidName() const override;
+
+  virtual Real molarMass() const override;
+
+  virtual Real T_from_p_h(Real pressure, Real enthalpy) const override;
+
+  Real T_from_p_rho(Real pressure, Real density) const;
+
+  virtual Real rho_from_p_T(Real pressure, Real temperature) const override;
+
+  virtual void rho_from_p_T(
```

Comment:
Let's group overloads with the same name without spacing
```suggestion
  virtual Real rho_from_p_T(Real pressure, Real temperature) const override;
  virtual void rho_from_p_T(
```

Reply:
addressed that